<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KEYWORD NEWS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Desktop/newspaper.png</normaloff>../../../../Desktop/newspaper.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>40</y>
      <width>771</width>
      <height>95</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KEYWORD NEWS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_greeting">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;나눔스퀘어&quot;;</string>
    </property>
    <property name="text">
     <string> 반갑습니다. keywordnews 프로그램에 오신 것을 환영합니다.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>260</y>
      <width>230</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
